.github/workflows/firebase-prod-merge.yml

```diff
@@ -0,0 +1,43 @@
+name: prod-deploy
+'on':
```

Comment:
remove quotes around `on`

Reply:
agree... that was how firebase created the action.. I was confused